AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -1226,4 +1212,18 @@ + (void)resetSharedInstance {
   sharedInstance = nil;
 }
 
+- (void)instanceDisableAutomaticCheckForUpdates {
+  @synchronized(self) {
+    if (self.started) {
+      MSLogError([MSDistribute logTag], @"Cannot disable automatic check for updates after Distribute is started.");
+      return;
+    }
+    self.automaticCheckForUpdatesDisabled = YES;
+  }
+}
+
++ (void)disableAutomaticCheckForUpdates {
```

Comment:
Please move it out of `#pragma mark - Private`. The place where was `+ (void)configure:` is OK